src/stores/BallotStore.js

```diff
@@ -22,7 +22,9 @@ class BallotStore {
 		2: 'VotingToChangeKeys',
 		3: 'VotingToChangeMinThreshold',
 		4: 'VotingToChangeProxy',
-		5: 'BallotsStorage'
+		5: 'BallotsStorage',
+		7: 'ValidatorMetadata',
```

Comment:
Why do we skip 6th index here?

Reply:
I took indexes from ProxyStorage.sol:

```
enum ContractTypes {
    Invalid,
    KeysManager,
    VotingToChangeKeys,
    VotingToChangeMinThreshold,
    VotingToChangeProxy,
    BallotsStorage, 
    PoaConsensus,
    ValidatorMetadata,
    ValidatorMetadataEternalStorage
}
```

i.e. 6th index is for `PoaConsensus`.